src/hotspot/share/gc/shared/taskTerminator.cpp

```diff
@@ -72,7 +72,7 @@ TaskTerminator::TaskTerminator(uint n_threads, TaskQueueSetSuper* queue_set) :
   _n_threads(n_threads),
   _queue_set(queue_set),
   _offered_termination(0),
-  _blocker(Mutex::leaf, "TaskTerminator", false, Monitor::_safepoint_check_never),
+  _blocker(Mutex::leaf, "TaskTerminator", Monitor::_safepoint_check_never),
```

Comment:
Is this lock even used by any JavaThreads? I'm guessing not - in which cases the always/never and allow/not-allow are immaterial, and the change is harmless.

Reply:
I see no evidence of this lock being taken by JavaThreads